### DISCUSSION TITLE
第3节课作业(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#228] [作业]
请提交作业的链接 Please submit a link to the assignment.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/677933496

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/3-InternLM-LangChain-93f6e16c46d44cfdbacd0032428332aa

**Comment by jayjayhust** [comment]
https://zhuanlan.zhihu.com/p/678561753

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/679036637
笔记和作业都是这个连接

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/677605223

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679576237

**Comment by YongXie66** [comment]
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/%E5%9F%BA%E4%BA%8E%20InternLM%20%E5%92%8C%20LangChain%20%E6%90%AD%E5%BB%BA%E4%BD%A0%E7%9A%84%E7%9F%A5%E8%AF%86%E5%BA%93.pdf

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/3
